05_syntax_lab/01.py

```diff
@@ -1,5 +1,29 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+#Week01_Exercise01_Assignment01

```

Comment:
Hi Nir and welcome aboard,

Please leave the top comments in the file (the ones inside """). They make it easier for everyone opening the file later to understand what's inside